content/tsconfig.json

```
@@ -1,7 +1,8 @@
 {
   "extends": "../tsconfig.json",
-  "include": ["src", "test"],
+  "include": ["src", "test", "./jest.*.ts"],
   "compilerOptions": {
+    "types": ["node", "jest"],
```

Types for "jest" should only be included in the `content/test/tsconfig.json`
```suggestion
    "types": ["node"],
```

---

Removed it. It was a way to avoid come clashes with Jasmine in the meantime but it is not necessary anymore.